source/plugins/ruby/KubernetesApiClient.rb

```diff
@@ -1382,14 +1382,16 @@ def getHpaOptimizedItem(resourceItem)
       return item
     end
 
-    def getPodReadyCondition(podStatusConditions)
+    def getPodReadyCondition(podStatusConditions, controllerKind, podStatus)
       podReadyCondition = false
       begin
         if !podStatusConditions.nil? && !podStatusConditions.empty?
           podStatusConditions.each do |condition|
             if condition["type"] == "Ready"
               if condition["status"].downcase == "true"
                 podReadyCondition = true
+              elsif isCompletedJobPod(controllerKind, podStatus)
```

Comment:
can we combine this and make this an or condition?

Reply:
> can we combine this and make this an or condition?

fixed.